### PR TITLE
Descriptions update

### PR DESCRIPTION
**Pull request by geramirez**
Makes ids, descriptions, and abbreviations to go under the "departments" level in the yaml files. Also, updates layer_with_usa_contacts.py and associated tests.

### CODE REVIEW

**Comment by khandelwal** [comment]
I think with this change it would be important to make sure the information that's changed is displayed correctly in our application. Did you get a chance to check that?

**Comment by geramirez** [comment]
@khandelwal , well it breaks everything starting with the abbreviation, which appear to be required at the top level. 

I went back and reversed the damage. Still, we should merge this PR because it adds 51 descriptions. Only 5 are missing now and I can add those manually.

**Comment by geramirez** [comment]
Only one missing description now; [Office of the Secretary of Defense and Joint Staff](http://www.dod.gov/pubs/foi/)

File: contacts/data/DOC.yaml [hunk 1]
```diff
@@ -880,5 +907,4 @@ phone: 202-482-3258
 public_liaison: 'Brenda Dolan, Phone: (202) 482-3258'
 service_center: 'Phone: (202) 482-3258'
 top_level: false
-usa_id: '52668'
```
Comment (khandelwal) [inline contacts/data/DOC.yaml:883]: How did the usa_id get removed? Did it go somewhere else?

**Comment by geramirez** [comment]
Ok, from now on all descriptions, ids, and abbreviations go to the top level first, anything left over goes to the department level.

File: contacts/data/Treasury.yaml [hunk 2]
```diff
@@ -255,64 +263,32 @@ departments:
   - Washington, DC 20227
   fax: 202-874-2391
   keywords:
-  - Accounting
```
Comment (khandelwal) [inline contacts/data/Treasury.yaml:258]: I'm sorry. I keep finding things in this pull request that make me uncomfortable. There is now this list of keywords that has been deleted. Where did it go? Is it not relevant anymore?

Comment (geramirez) [inline contacts/data/Treasury.yaml:258]: Keywords keep on changing when I run keywords_from_fr.py, I'll run everything again and see what happens.

Comment (geramirez) [inline contacts/data/Treasury.yaml:258]: Just updated again

**Comment by geramirez** [comment]
@khandelwal, ok just updated and ran layer_with_usa_contacts.py a couple times to make sure nothing changed.

**Pull request merged**